Sources/Components/SelectionBoxes/Demo/CheckboxDemoView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+class CheckboxDemoView: UIView {
+    let strings = [
+        "Mistanke om svindel",
+        "Regelbrudd",
+        "Forhandler opptrer som privat",
+    ]
+
+    lazy var checkbox: Checkbox = {
+        let box = Checkbox(strings: self.strings)
```

Comment:
I think we are going to see what things we need when we implement this on the FINN app but we might need more than title strings then we might need to look into the DataSource pattern used in other pars of FinniversKit.